### PR TITLE
Update dependencies of plugins

### PR DESCRIPTION
**Pull request by goderbauer**
Fixes https://github.com/flutter/flutter/issues/10749.

Some of these changes are breaking changes as noted in the CHANGELOGs. I am preparing an email [1] that will be send out to flutter-users before publishing the updated plugins to pub. No action is required for new flutter apps (after https://github.com/flutter/flutter/pull/10842 is merged). This is only a breaking change because of https://github.com/flutter/flutter/issues/9881.

Plugin users will also have to make sure that they update their "Google Repository" version to 54 (or newer) in the Android SDK manager.

[1] https://docs.google.com/document/d/1B9RZ-PpdFKdgUvLxu1mvBxwG3ytDVJo0GFxCZVMyu0E/edit

### CODE REVIEW

**Comment by collinjackson** [comment]
I wonder if we should bump the version to 0.1.x for breaking changes so people on ^0.0.x won't get the upgrade by default.

File: .travis.yml [hunk 1]
```diff
@@ -70,15 +56,30 @@ matrix:
           sources:
             - ubuntu-toolchain-r-test # if we don't specify this, the libstdc++6 we get is the wrong version
           packages:
+            - lib32stdc++6 # https://github.com/flutter/flutter/issues/6207
```
Comment (goderbauer) [inline .travis.yml]: Please ignore the changes in this file, I've send out these changes in a separate PR (https://github.com/flutter/plugins/pull/145) and will rebase this PR on top of the other.

**Comment by goderbauer** [comment]
Rebased & bumped versions. PTAL @collinjackson
